services/gateway/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -ex
+
+JWT_SOURCE_64=$(base64 < /openssl-certs/madoc.pub);
+
+IFS=',' read -ra ADDR <<< "$JWT_SERVICES"
+for i in "${ADDR[@]}"; do
+    echo $(cat "./apps/$i.json" | jq --arg value "$JWT_SOURCE_64" '.jwt_source = $value') > "./apps/$i.json"
+done
+
+
+./entrypoint.sh
```

Comment:
@donaldgray @garyttierney This is a limitation of the way you configure tyk to verify tokens that it doesn't issue. You have put the public key into the configuration. To make this dynamic, I'm just setting this in the entrypoint on a set of configurations passed in. 

There's a small pile up of dynamic things in these configurations. I'm considering creating a lighter, simple JSON format for services to describe some of this. That way, for a specific configuration, like Ghent, the actual tyk configurations can be generated correctly (with proxy + host set up, along with this JWT). I'm picturing this generation being part of the config repo – not dynamic. So they can still be changed and customised as much as needed.

Does need some thought though, it's not the easiest to configure. Similar issue with expanding the shared postgres database, where N number of services want a database at runtime.